XML/TestCases/StressTests.xml

```diff
@@ -32,8 +32,8 @@
 	</test>
 		<test>
 		<testName>STRESSTEST-RELOAD-MODULES-UP</testName>
-		<testScript>BVT-CORE-RELOAD-MODULES.ps1</testScript>
```

Comment:
I think the TC name is not clear enough for new comers. Can you change the test name to something like 'RELOAD-LIS-MODULES' or 'RELOAD-LIS-KERNEL-MODULES'